src/pages/Photos/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import GalleryList from "../../components/Banner/GalleryList";
+
+const Photos = ({ searchTerm }) => {
+  return <GalleryList searchTerm={searchTerm} />;
+};
+
+export default Photos;
```

Comment:
Do API calls here and pass data as props to galleryList component. Do the same for the favorite page.